src/components/Questionnaire/QuestionnaireForm.tsx

```
@@ -712,39 +721,50 @@ export function QuestionnaireForm({
-                  </button>
-                ))}
-            </div>
-          </div>
+          <>
+            {encounterId === "preview" ? (
+              <QuestionnaireNavigation
+                questionnaire={form.questionnaire}
+                scrollToQuestion={(questionId?: string) => {
+                  scrollToQuestion(form.questionnaire.id, questionId);
+                }}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)